dlc-test/src/test/scala/org/bitcoins/dlc/testgen/TestVectorGen.scala

```diff
@@ -36,6 +36,12 @@ trait TestVectorGen[T <: TestVector, Input] {
   }
 
   def readFromDefaultTestFile(): JsResult[Vector[T]] = {
+    // If we are missing the test file, generate it
+    if (!defaultTestFile.canRead) {
+      val gen = generateAndWriteTestVectors()
+      Await.result(gen, 60.seconds)
```

Comment:
Any reason to not propagate the Future?

Reply:
Would have been required a much larger refactor, probably better for a separate PR